### PR TITLE
Added ingestion documentation

### PR DESCRIPTION
**Pull request by Davleen14**
Added ingestion documentation

### CODE REVIEW

File: docs/ingestion/elastic-search.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Elastic Search
+
```
Comment (sunitakawane) [inline docs/ingestion/elastic-search.md:2]: Add a line something like `assuming your siglens instance is running on localhost with ingestPort 8081`

File: docs/ingestion/loki.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    labels:
+      job: varlogs
+      __path__: /var/log/*log
+```
+#### Run the binary:
+- ./promtail-darwin-amd64 -config.file=promtail-config.yaml
+
+Promtail will start ingesting the logs from the /var/log folder and then push them to siglens
+
+For promtail ingestion to work, we have added proto files in the siglens repo in loki pkg. These proto files are used to generate the go files which are used by promtail to push the logs to siglens. The source protos are `push.proto, stats.proto and logproto.proto`. The *.pb.go were generated using this pkg `google.golang.org/protobuf/cmd/protoc-gen-go`` and running command `protoc --go_out=. path/to/your/proto/file.proto.`
```
Comment (sunitakawane) [inline docs/ingestion/loki.md:31]: The last paragraph is not needed here.

File: docs/ingestion/vector.md [hunk 3]
```diff
@@ -2,4 +2,33 @@
 sidebar_position: 6
 ---
 
-# Vector
\ No newline at end of file
+# Vector
+
+- Similar to FluentD, SigScalr is compatible with the Elastic plugin for vector. Install vector and start it.
```
Comment (sunitakawane) [inline docs/ingestion/vector.md:7]: replace sigscalr with siglens

File: docs/ingestion/loki.md (hunk 2 above)
Comment (sunitakawane) [inline docs/ingestion/loki.md]: remove this para , starting with `For promtail ingestion to work, we have added proto files in the siglens repo in loki pkg....`

**Review by sunitakawane — APPROVED** [review]

**Comment by sunitakawane** [comment]
@sonamgupta21 can you also take a look, to see if any formatting changes are needed

**Review by sonamgupta21 — APPROVED** [review]
lgtm.

**Pull request merged**